src/launch/policy.c

```diff
@@ -999,64 +1046,42 @@ int policy_export(Policy *policy, sd_bus_message *m) {
                         return error_origin(r);
         }
 
-        r = sd_bus_message_open_container(m, 'r', "buu(" POLICY_T_BATCH ")");
-        if (r < 0)
-                return error_origin(r);
+        {
+                uid_t prev_uid = 0;
 
-        r = sd_bus_message_append(m, "buu", false, 0, SYSTEMUIDMAX);
-        if (r < 0)
-                return error_origin(r);
+                if (n_at_console_uids > 0) {
```

Comment:
Stupid qsort() does not work with NULL+0, right? Meh...

Reply:
As a wise man (you) pointed out in private chat, NULL+0 is actually fine: https://sourceware.org/git/?p=glibc.git;a=blob;f=stdlib/qsort.c;h=264a06b8a924a1627b3c0fd507a3e2ca38dbc8a0;hb=HEAD#l98 I'll simplify.